docs/usage/9.md

```diff
@@ -0,0 +1,20 @@
+Manual Rollback Procedure
+=========================
+If a move operation was performed on an application requiring quiescing, care must be taken to prevent data corruption. Ensure that applications on the destination cluster aren't accessing data on the PVC before trying to bring your applications back up on the source cluster. In a move operation, the migration plugin ensures that the retention policy for bound PVs is 'Retain' to prevent data loss when an application is cleaned up on the source cluster.
```

Comment:
Is it intended to repeat this text here and Storage section?
May be better to remove it here and keep once under Storage?

Consider adding a sentence to summarize what 'Manual Rollback' is for, to help restore an application on source cluster if it was quiesced and/or PVs were moved.  For applications that weren't quiesced and did not move a PV, manual rollback is not required as the source application is still running and any data if handled would be a copy to destination.

Reply:
I just saw the original text and like it, guess this just an accident of deleting the original and duplicating the below.

"If a migration operation was performed without quiescing the application these steps should not be necessary since the application will be running in place on the source cluster. In a case where a quiesced migration operation was performed and rollback is desired for due to failure or some other reason follow the steps below."

Reply:
Ya, just an accident. I think I fixed my screw ups and managed to apply the changes Derek suggested.